packages/g/gammaray/pspec_x86_64.xml

```diff
@@ -0,0 +1,292 @@
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_quickinspector.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_quickwidgetsupport.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_sceneinspector_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_scriptenginedebugger_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_signalmonitor.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_statemachineviewer_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_styleinspector_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_sysinfo_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_textdocumentinspector.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_timertop_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_translatorinspector.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_webinspector_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_widgetinspector_plugin.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/gammaray_wlcompositorinspector.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/libgammaray_widget_export_actions.so</Path>
+            <Path fileType="library">/usr/lib64/gammaray/3.0/qt5_15-x86_64/target/position/libgammaray_geopositioninfosource.so</Path>
+            <Path fileType="library">/usr/lib64/libgammaray_common-qt5_15-x86_64.so.3.0.0</Path>
+            <Path fileType="library">/usr/lib64/libgammaray_core-qt5_15-x86_64.so.3.0.0</Path>
+            <Path fileType="library">/usr/lib64/libgammaray_kitemmodels-qt5_15-x86_64.so.3.0.0</Path>
+        </Files>
+    </Package>
+    <Package>
+        <Name>gammaray-probe-qt5-devel</Name>
+        <Summary xml:lang="en">Development files for gammaray-probe-qt5</Summary>
+        <Description xml:lang="en">Provides a Qt 5 probe for GammaRay that allows introspecting Qt 5 applications.</Description>
+        <PartOf>programming.devel</PartOf>
+        <RuntimeDependencies>
+            <Dependency release="1">gammaray-probe-qt5</Dependency>
+        </RuntimeDependencies>
+        <Files>
+            <Path fileType="library">/usr/lib64/libgammaray_common-qt5_15-x86_64.so</Path>
+            <Path fileType="library">/usr/lib64/libgammaray_core-qt5_15-x86_64.so</Path>
+            <Path fileType="library">/usr/lib64/libgammaray_kitemmodels-qt5_15-x86_64.so</Path>
```

Comment:
TBH if this is all that's in gammaray-probe-qt5-devel I'd probably just move them into gammaray-probe-qt5, or delete them. They're basically useless